Renaming `owner` role name to `admin` (#2125)
codemagic-ci-cd/codemagic-docs: content/yaml-running-builds/starting-builds-automatically.md

> When using the Flutter workflow editor, by clicking **Update Webhook** under the Webhooks tab, Codemagic automatically adds webhooks to the repositories added via GitHub app or from GitHub, GitLab or Bitbucket via the OAuth integration after you have enabled any of the triggers in this section. For repositories added via SSH or HTTP/HTTPS, or if you are configuring your builds using `codemagic.yaml`, you would have to [set up webhooks manually](../building/webhooks). Note that webhook triggers might not be supported for all repository providers.

Reviewer: The whole Flutter workflow editor section seems irrelevant as we have a separate page about build triggers for workflow editor

Author: Correct, I removed it but kept the repos that have been added by SSH or for the yaml workflows section.

{{<notebox>}}

> **Note:** Webhooks can be clicked to update by the team admin who connected their repository in the Codemagic UI.

Reviewer: ```suggestion
**Note:** The team admin who added the repository can update the webhook by clicking the **Update webhook** button in the Codemagic UI.
```
